corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap3/crud_paginated_list.js

```diff
@@ -143,16 +143,15 @@ hqDefine("hqwebapp/js/bootstrap3/crud_paginated_list", [
         };
 
         self.initCreateForm = function () {
-            var $createForm = $("#create-item-form");
+            const $createForm = $("#create-item-form");
             $createForm.submit(function (e) {
                 e.preventDefault();
-                $createForm.ajaxSubmit({
-                    url: "",
-                    type: 'post',
+                let formData = new FormData($createForm.get(0));
+                formData.set("action", "create");
+                $.ajax({
+                    method: 'POST',
                     dataType: 'json',
```

Comment:
I don't quite follow why we keep `dataType: 'json'` in this request but remove it in the other modified request in this file? Same goes for the b5 version.

Reply:
Those are spurious changes, I must have made them while troubleshooting and then not changed them back. Reading the docs now, it'd be fine to leave off `dataType` altogether in this code (for a default value, the [docs](https://api.jquery.com/jQuery.ajax/) say it makes an intelligent guess).